src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="personalInformation">
+        <legend>Personal information</legend>
+        <label class="form-field">
+        Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          required>
+
+        </label>
```

Comment:
fix formatting for all inputs

```suggestion
        <label class="form-field">
          Surname:
          <input
            type="text"
            name="surname"
            autocomplete="off"
            required
          >
        </label>
```